.github/workflows/base-images.yml

```diff
@@ -123,14 +123,122 @@ jobs:
       - name: Build ubuntu Base Image
         uses: docker/build-push-action@v3
         with:
-          file: .github/docker-images/base-images/ubuntu/Dockerfile
+          file: .github/docker-images/base-images/device-client/ubuntu/Dockerfile
           context: .
           target: base
           push: true
           tags: |
             public.ecr.aws/${{ env.ECR_BASE_REPO }}:armv7-ubuntu-latest
           platforms: linux/arm/v7
-  build-base-docker-image-amazonlinux-amd64:
+  build-integration-test-base-docker-image-ubuntu-x86_64:
+    runs-on: ubuntu-latest
+    permissions:
+      id-token: write
+      contents: read
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.ECR_USER_AWS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.ECR_USER_AWS_KEY_SECRET }}
+          aws-region: us-east-1
+      - name: Login to ECR
+        run: aws ecr-public get-login-password --region us-east-1 | docker login --username AWS --password-stdin public.ecr.aws
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: public.ecr.aws/${{ env.ECR_INTEG_BASE_REPO }}
```

Comment:
are these lines still needed? I notice it exists in other jobs as well but what does this actually do?

Reply:
It just uploads metadata about the image to the repo in case it is needed for some reason.